lzy/worker/src/main/java/ai/lzy/worker/env/SimplePythonEnvironment.java

```diff
@@ -0,0 +1,93 @@
+package ai.lzy.worker.env;
+
+import ai.lzy.v1.common.LME;
+import ai.lzy.worker.StreamQueue;
+import org.apache.commons.io.IOUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class SimplePythonEnvironment implements AuxEnvironment {
```

Comment:
let's avoid such a general words as `simple`, `common`, `complex`, ... :)

here you may wish to use `PurePythonEnv`, for example

Reply:
or `PlainPythonEnv` as you use in logs below